### DISCUSSION TITLE
Allow negative numbers

### DISCUSSION THREAD

**Discussion by daveldt** [iSoron/uhabits#2067] [Feature Requests]
Hey it's possible to add negative numbers?
For example to track my bank account 
I have income and expens.
Tue -100$
Wed +300$
Thu +400$
Fri   -500$

Thanks 
This app is amazing anyway!! 
Bye

**Comment by adrienbeau** [comment]
You need a personal finance app for that.

And maybe set-up a habit to use it regularly, is that is an issue for you.

**Comment by daveldt** [comment]
That's ok! Thank you!

Inviato da Outlook per Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by herrbrixter** [comment]
Duplicate of #1385
